packages/backend/src/database/entities/spaces.ts

```diff
@@ -10,11 +10,12 @@ export type DbSpace = {
     organization_uuid: string;
     created_by_user_id?: number;
     search_vector: string;
+    slug: string;
```

Comment:
I made changes to all db models and added slug on creation, I didn't add it to the `Space/Chart/Dashboard` model because we don't expose this on the API or anywhere else yet.